vignettes/safestats-vignette.Rmd

```diff
@@ -482,8 +480,8 @@ sum(sValuesBatch1To2 > 1/alpha)
 
 an additional 135 null rejections (93.1% of 145 experiments). This brings up the total number of null rejections to 990 out of 1,000 experiments. In this case, a null rejection is correct, since the data were generated with a true effect that was equal to deltaMin. 
 
-### S-values for replication studies
-It is not always appropriate to combine data sets. In particular, for replication attempts where the original experiment is done on a different population. In that case, one can still easily do safe inference by multiplying the s-values computed over each data set separatedly. This procedure also conserves the $\alpha$-level, as we show below. 
+### b. Optional continuation through replication studies
```

Comment:
I'm not sure whether we should use "b.", as I missed out on seeing "a.". The title without "b." might be already good enough.

Reply:
see above: "a. Optional continuation by _extending_ ... "